NAMESPACE

```diff
@@ -21,6 +21,7 @@ importFrom(progress,progress_bar)
 importFrom(quarto,quarto_render)
 importFrom(renv,activate)
 importFrom(renv,restore)
+importFrom(rstudioapi,selectDirectory)
```

Comment:
I do not know if it is worth it to add a new dependency for this task (especially because of the RStudio requirement). Perhaps we can just force the user to define the destination file.

Reply:
True, there's no point of adding a new dependency. We can keep it as simple and ask user to define the path for file.